tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -1067,7 +1076,7 @@ describe(__filename, () => {
         url: `/${lang}/${clientApp}/user/${anotherUserId}/`,
       }),
     );
-    sinon.assert.calledOnce(dispatchSpy);
+    sinon.assert.calledTwice(dispatchSpy);
```

Comment:
As above, please add a comment explaining why we expect this to be called twice.